omp/test/solver/lower_trs_kernels.cpp

```diff
@@ -93,25 +93,17 @@ class LowerTrs : public ::testing::Test {
     {
         b = gen_mtx(m, n);
         x = gen_mtx(m, n);
-        t_b = Mtx::create(ref);
-        t_x = Mtx::create(ref);
-        t_b->copy_from(b.get());
-        t_x->copy_from(x.get());
-        d_b = Mtx::create(omp);
-        d_b->copy_from(b.get());
-        d_x = Mtx::create(omp);
-        d_x->copy_from(x.get());
-        dt_b = Mtx::create(omp);
-        dt_b->copy_from(b.get());
-        dt_x = Mtx::create(omp);
-        dt_x->copy_from(x.get());
+        t_b = b->clone();
+        t_x = x->clone();
```

Comment:
just curios, the cuda and hip do not use t_b and t_x?

Reply:
Yes, they don't use the individual kernels, but the `core` interface.